### DISCUSSION TITLE
Qtile is broke with the latest upgrade.

### DISCUSSION THREAD

**Discussion by dm7r33** [qtile/qtile#4961] [Q&A] [answered]
Qtile is broke with the latest upgrade.

```
> qtile --verison
 0.27.1.dev0+g17cc04b3.d20240714
> pacman -Q qtile
qtile 0.27.0-1
```

My monitor configuration is messed up on startup, but as soon as i reload my config it's all fine. So it seems like there is some sort of race condition that got introduced with the newest update and is not fixed in the current config provided by arcolinux. I tried it with the configuration in /etc/skel (from the arcolinux repository) with my xrandr script in the autostart.sh script, but that didn't change anything. Any ideas what might be the issue here?

FYI: arcolinux is an archlinux distribuation

All the best
diemytree

**Comment by elParaguayo** [comment]
Should be fixed in 0.28.0

**Reply by dm7r33** [reply]
Can you explain to me what the issue is? thanks for the fast reply.

**Reply by elParaguayo** [reply, marked as answer]
#4936 is another example. We changed something to fix one problem but it unexpectedly caused others. We've reverted that change for now.
